.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "loader"]
 	path = external/OpenCL-ICD-Loader
-	url = https://github.com/KhronosGroup/OpenCL-ICD-Loader.git
+	url = https://github.com/StreamHPC/OpenCL-ICD-Loader.git
```

Comment:
https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/226 has been merged, can we change this line back?

Reply:
Sure, good catch! Should be updated now